gnome-extra/gnome-calendar/gnome-calendar-43.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome.org gnome2-utils meson virtualx xdg
+
+DESCRIPTION="Manage your online calendars with simple and modern interface"
+HOMEPAGE="https://wiki.gnome.org/Apps/Calendar"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+
+DEPEND="
+	>=dev-libs/libical-1.0.1:0=
+	>=gnome-base/gsettings-desktop-schemas-3.21.2
+	>=gnome-extra/evolution-data-server-3.46.0:=[gtk]
```

Comment:
In practice the version doesn't matter here, but we should keep it in sync with `meson.build` to make recognizing updates easier.

````diff
-#libedataserverui_dep = dependency('libedataserverui-1.2', version: '>= 3.17.1')
-libedataserver_dep = dependency('libedataserver-1.2', version: '>= 3.17.1')
-libecal_dep = dependency('libecal-2.0', version: '>= 3.33.2')
-libsoup_dep = dependency('libsoup-2.4')
-libadwaita_dep = dependency('libadwaita-1', version: '>= 1.0.0')
+libedataserverui_dep = dependency('libedataserverui4-1.0', version: '>= 3.45.1')
+libedataserver_dep = dependency('libedataserver-1.2', version: '>= 3.45.1')
+libecal_dep = dependency('libecal-2.0', version: '>= 3.45.1')
+libsoup_dep = dependency('libsoup-3.0')
+libadwaita_dep = dependency('libadwaita-1', version: '>= 1.2.alpha')
````

That is, change it to `3.45.1`.